### DISCUSSION TITLE
Blank page after successful Vercel deployment

### DISCUSSION THREAD

**Discussion by vinhkoi** [vercel/community#7372] [Help] [answered]
### Summary

After I successfully deloy the website, when accessing it via https://laravel-vuejs.vercel.app/ it was an empty website with nothing even though locally it worked fine.I use laravel and vuejs to connect frontend and backend using inertia

### Example

https://laravel-vuejs.vercel.app/

### Steps to Reproduce

Here is my project github link: https://github.com/vinhkoi/laravel-vuejs/tree/test
I hope someone can help me solve this problem, I tried ngrok and got the same error when accessing from another device.
I checked the build log and there were no errors reported
![image](https://github.com/vercel/community/assets/99164411/a1348ca8-6075-4da9-93d7-d01a6d711808)

![image](https://github.com/vercel/community/assets/99164411/9a49e4a6-8f72-4ac4-8872-3937defe1d25)

**Comment by pawlean** [comment]
Hi @vinhkoi!

Apologises, I wasn't sure what you mean here. Are you trying to delete your app?

**Reply by vinhkoi** [reply]
No no,I deployed my project using Vercel, but when I access the deployed website, it's a blank page.I want to fix it,

**Reply by pawlean** [reply]
Have you tried configuring you Root Directory? From [relevant documentation](https://vercel.com/docs/deployments/configure-a-build#root-directory):

> In some projects, the top-level directory of the repository may not be the root directory of the app you’d like to build. For example, your repository might have a front-end directory containing a stand-alone [Next.js](https://nextjs.org/) app.

**Reply by vinhkoi** [reply]
I tested it, no difference
![image](https://github.com/vercel/community/assets/99164411/cd35aab2-ec59-415b-af85-81eebe2f0d75)
![image](https://github.com/vercel/community/assets/99164411/512b0207-cf09-4f87-9bec-66f87b1545b5)
I think there is no problem with the directory, I tried checking the build log with requests and there is still a 200 response but nothing is displayed.

**Reply by vinhkoi** [reply]
![image](https://github.com/vercel/community/assets/99164411/b5a1dd98-357d-4c44-8add-bb8d2ecc79c2)

**Reply by QuiiBz** [reply]
👋 Looking at the console when accessing https://laravel-vuejs.vercel.app, there are a few errors of JavaScript files failing to load:
<img width="1469" alt="Screenshot 2024-07-05 at 16 53 34" src="https://github.com/vercel/community/assets/43268759/04765e1b-da78-4411-8091-7367d53c81e5">

For example, `https://laravel-vuejs.vercel.app/build/assets/app-4cc3f654.js` returns some HTML, which is probably because the `/build/assets/app-4cc3f654.js` file doesn't exist in your deployment and fallbacks to calling the Serverless Function (as you can also see in your logs):
<img width="745" alt="Screenshot 2024-07-05 at 16 55 39" src="https://github.com/vercel/community/assets/43268759/1e68a520-a9d3-439d-983f-27d4ece86abc">

Please make sure the `Output directory` in the `General` tab of your project settings specifies the correct folder, and that you are loading those JavaScript and CSS files with the correct path (I'm wondering if it should be e.g. `/assets/app-4cc3f654.js` instead of `/build/assets/app-4cc3f654.js`?)

**Reply by vinhkoi** [reply]
![image](https://github.com/vercel/community/assets/99164411/61aa54a9-1dd2-49f7-9a69-c410995bf79d)
![image](https://github.com/vercel/community/assets/99164411/bd38e029-879b-41b9-8bc9-a315463d2351)
![image](https://github.com/vercel/community/assets/99164411/cf08db18-6f86-4971-8b88-d78744d13497)
Thank you for your reply I tried to check,this is my configuration.Do you need any more information?

**Reply by QuiiBz** [reply]
Thanks for sharing more details. Looking at your screenshots, my assumption was correct: your JavaScript and CSS assets are available in the `assets` folder, but your HTML targets `build/assets`. You should be able to fix it by updating the "Output directory" to be `public` instead of `public/build`.

**Reply by vinhkoi** [reply]
![image](https://github.com/vercel/community/assets/99164411/ae0af3b5-fa9b-4c04-a6be-4bef71104b2b)
![image](https://github.com/vercel/community/assets/99164411/6f1681a1-159a-45a2-9f1b-19f48e9c21d7)
![image](https://github.com/vercel/community/assets/99164411/71d0b7c7-87ae-40db-8ee6-6243e794f873)
Thank you for your help, I tried changing the output directory but I still get the same result @@ can you take a look at my github maybe I chose the wrong directory?
https://github.com/vinhkoi/laravel-vuejs/tree/test

**Reply by QuiiBz** [reply, marked as answer]
Could you try to add a new property in the `routes` configuration of your `vercel.json`?

```json
{
  "src": "/build/(.*)",
  "dest": "/build/$1"
},
```

**Reply by vinhkoi** [reply]
Thank you very much I got it running, it finally displays the content even though the interface is broken but it finally works
![image](https://github.com/vercel/community/assets/99164411/0a4fe9e1-98c8-4ba3-9a10-77757d815943)
Can I ask a little more? If the interface is broken, is it possible that the css file was not imported correctly?

[a comment by an automated account is not shown]
